.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -70,7 +70,7 @@ verify features introduced or bugs fixed in this PR.
 
  -->
 
-## Where any tests migrated to React Testing Library?
+## Were any tests migrated to React Testing Library?
```

Comment:
Oh my goodness, thank you! This typo hollers in my head every time I submit a PR 🤣